var/spack/repos/builtin/packages/symengine/package.py

```diff
@@ -35,7 +36,9 @@ class Symengine(Package):
     version('0.1.0', '41ad7daed61fc5a77c285eb6c7303425')
     version('develop', git='https://github.com/symengine/symengine.git')
 
-    variant('flint',        default=True,
+    variant('boost',        default=True,
```

Comment:
@isuruf  Is `mpc` a strict requirement for `symengine`? I didn't think it was, but even with `~mpc` (a requirement for `+boostmp`) `symengine` fails to build as it checks for `mpc` using  `cmake/FindMPC.cmake`. 
[Log attached.](https://github.com/LLNL/spack/files/725922/spack-build.out.txt)
For the record, here's the DAG with the updated package as well
```bash
$ spack spec symengine@develop +boostmp
Concretized
--------------------------------
symengine@develop%clang@7.3.0-apple+boostmp~flint+mpc+mpfr~openmp~piranha+shared+thread_safe arch=darwin-elcapitan-x86_64
    ^boost@1.62.0%clang@7.3.0-apple+atomic+chrono+date_time~debug+filesystem~graph~icu+iostreams+locale+log+math+mpi+multithreaded+program_options+python+random+regex+serialization+shared+signals~singlethreaded+system~taggedlayout+test+thread+timer+wave arch=darwin-elcapitan-x86_64
        ^bzip2@1.0.6%clang@7.3.0-apple arch=darwin-elcapitan-x86_64
        ^openmpi@2.0.1%clang@7.3.0-apple~java~mxm~pmi~psm~psm2~slurm~sqlite3~thread_multiple~tm~verbs+vt arch=darwin-elcapitan-x86_64
            ^hwloc@1.11.4%clang@7.3.0-apple arch=darwin-elcapitan-x86_64
                ^libpciaccess@0.13.4%clang@7.3.0-apple arch=darwin-elcapitan-x86_64
                    ^libtool@2.4.6%clang@7.3.0-apple arch=darwin-elcapitan-x86_64
                        ^m4@1.4.17%clang@7.3.0-apple+sigsegv arch=darwin-elcapitan-x86_64
                            ^libsigsegv@2.10%clang@7.3.0-apple arch=darwin-elcapitan-x86_64
                    ^pkg-config@0.29.1%clang@7.3.0-apple+internal_glib arch=darwin-elcapitan-x86_64
                    ^util-macros@1.19.0%clang@7.3.0-apple arch=darwin-elcapitan-x86_64
        ^python@2.7.12%clang@7.3.0-apple~tk~ucs4 arch=darwin-elcapitan-x86_64
            ^ncurses@6.0%clang@7.3.0-apple arch=darwin-elcapitan-x86_64
            ^openssl@1.0.2j%clang@7.3.0-apple arch=darwin-elcapitan-x86_64
                ^zlib@1.2.8%clang@7.3.0-apple+pic arch=darwin-elcapitan-x86_64
            ^readline@6.3%clang@7.3.0-apple arch=darwin-elcapitan-x86_64
            ^sqlite@3.8.5%clang@7.3.0-apple arch=darwin-elcapitan-x86_64
    ^cmake@3.6.1%clang@7.3.0-apple~doc+ncurses+openssl+ownlibs~qt arch=darwin-elcapitan-x86_64
```